packages/stable/src/api/userProfiles/profilesApi.ts

```diff
@@ -0,0 +1,64 @@
+// Copyright 2023 Cognite AS
+
+import { BaseResourceAPI } from '@cognite/sdk-core';
+import {
+  UserProfileItem,
+  UserIdentifier,
+  UserProfilesSearchRequest,
+  UserProfilesListResponse,
+} from '../../types';
+
+export class ProfilesAPI extends BaseResourceAPI<UserProfileItem> {
+  /**
+   * [Get the user profile of the principal issuing the request](https://developer.cognite.com/api#tag/User-profiles/operation/getRequesterUserProfile)
+   *
+   * ```js
+   * const response = await client.profiles.me();
+   * ```
+   */
+  public me = async (): Promise<UserProfileItem> => {
+    const path = this.url('me');
+
+    const response = await this.get<UserProfileItem>(path);
+    return this.addToMapAndReturn(response.data, response);
+  };
+
+  /**
+   * [List all user profiles](https://developer.cognite.com/api#tag/User-profiles/operation/listUserProfiles)
+   *
+   * ```js
+   * const response = await client.profiles.list({ limit: 1000, nextCursor: 'abc' });
+   * ```
+   */
+  public list = (scope?: {
+    limit?: number;
+    cursor?: string;
+  }): UserProfilesListResponse => {
+    return super.listEndpoint(this.callListEndpointWithGet, scope);
+  };
```

Comment:
you could reuse the common type
```suggestion
  public list = (scope?: FilterQuery): UserProfilesListResponse => {
    return this.listEndpoint(this.callListEndpointWithGet, scope);
  };
```

Reply:
ah nice